container-storage-setup.sh

```diff
@@ -624,6 +624,15 @@ is_block_dev_partition() {
     return 0
   fi
 
+  # This will check if a loop/nvme device is passed and is already partitioned.
+  if ! devicePartition=$(lsblk -npls -o NAME ${bdev}|tail -n +2); then
```

Comment:
how about calling it "devparent". And also make it `local` variable to funciton.